frontend/src/lib/components/Users/UserTable.svelte

```diff
@@ -4,7 +4,7 @@
   import { AdminIcon, Icon } from '$lib/icons';
   import { createEventDispatcher } from 'svelte';
   import Dropdown from '$lib/components/Dropdown.svelte';
-  import type { User } from './+page';
+  import type { User } from '../../../routes/(authenticated)/admin/+page';
```

Comment:
This isn't great, because if the component gets moved then this path won't have the same number of `..` in it. It would be better to move the definition of the `User` type out of the admin page and into the UserTable component, then have the admin page import the definition from the component (where you can use the `$lib` syntax to avoid any `../../..` nonsense). However, I won't ask you to do that as it would touch a lot of files; I'll create an issue to track that that work needs doing, and I'll do it after this PR is merged.